app/styles/users.scss

```diff
@@ -0,0 +1,18 @@
+.resultslist-header {
```

Comment:
I moved the file to components folder and nested the code under `#users` section.